### PR TITLE
0.2 Release

### PR DESCRIPTION
**Pull request by greenmustard**
0.2 Release Notes:

- Asset updates
-- Safe/Minimum following zone
-- Secondary vehicle is now white
-- Lane change
- Added parameterization section to scenarios readme
- New ID naming system for all scenarios
-- Added Scenarios IDs in readme
-- Renamed existing scenarios with new system
-- Action IDs key for every category for when actor's action is out of the ordinary/cannot be deduced
- Added scenarios in:
-- Car Following
-- Intersections

### CODE REVIEW

File: docs/scenarios/README.md [hunk 1]
```diff
@@ -20,9 +19,48 @@ Scenarios are divided into one or more steps. Each step has an image, scenario d
...
+
+## Parameterization
+
+Simulation allows us to take a single scenario and change the value of key variables. This allows to quickly produce a comprehensive suite of testing scenarios.
+
+In most of our scenarios, speed and/or distance of Ego and actors are parameterized.
+
 ## Behavioral Competencies
 
-Across the industry, lots of amazing work has already been done in defining standardized behavioral competencies for the safe operation of an AV. A subset of these competencies act as the foundation of OAS and are gathered from various public agencies and industry leaders.
```
Comment (cuffney) [inline docs/scenarios/README.md]: remove "lots of"

File: docs/scenarios/car-following.md [hunk 2]
```diff
@@ -9,34 +9,69 @@ Work In Progress
...
+
+
+## Lead vehicle cuts in
+**2-2-XX-CF-STR-CAR:Pa>E:03**
+
+A secondary vehicle cuts in ahead of Ego
+
+|    |                                    |          |
+| -- | ---------------------------------- | -------- |
+|  A  | ![Vehicle cut in](./images/CF-04-A.png) | **Scenario** <br> Secondary vehicle cuts in ahead of Ego inside the safe following distance. <br><br> **Expected Result** <br> Ego slows down and then follows lead vehicle.  |
```
Comment (cuffney) [inline docs/scenarios/car-following.md]: remove "then"

File: docs/scenarios/crosswalks.md [hunk 3]
```diff
@@ -128,13 +128,12 @@ Ego approaches crosswalk at a 3 way stop and pedestrian is crossing in lane of t
 |  B  | ![3 way stop and pedestrian in lane-B](./images/CW-11-B.png) | **Scenario** <br> Pedestrian exits crosswalk. <br><br> **Expected Result** <br> Ego waits until the pedestrian is physically clear of the crosswalk and then proceeds.  |
```
Comment (cuffney) [inline docs/scenarios/crosswalks.md]: remove "then"

File: docs/scenarios/README.md [hunk 4]
```diff
@@ -20,9 +19,48 @@ Scenarios are divided into one or more steps. Each step has an image, scenario d
...
+
+* **scenario category:** The high-level category
+<br>Car following `CF`, crosswalks `CW`, cul-de-sacs `CDS`, intersections `I`, pedestrians in road `PIR`, reversing vehicles `RV`, speed limits `SL`, vehicles in roadway `VR`, etc
+
+* **Ego action:** What Ego will be doing in the scenario
+<br>Straight `STR`, left `L`, right `R`, u-turn `U`, etc
+
+* **other actors:** The other actors in the scenario: actors' starting > ending positions (if applicable): actor actions (if applicable)
+<br>`actor:start>end:action`  
+<br>**actor:** Car `CAR`, bus `BUS`, bicycle `BIKE`, motorcycle `M`, golf cart `GC`, pedestrian `PED`, etc <br>
```
Comment (Eric-Gonzalez) [inline docs/scenarios/README.md]: How are multiple actors represented? How are multiple actors in different locations represented?

**Comment by Eric-Gonzalez** [comment]
@greenmustard Looking good, but I have a couple of questions and comments.

File: docs/scenarios/README.md (hunk 4 above)
Comment (greenmustard) [reply docs/scenarios/README.md]: Currently added to the end of the scenario ID, so every scenario ID has at minimum 6 sections (and if it has more than 6 sections, all the additional sections are just additional actors). Sort order for actors is based on the order in which Ego "encounters" the actors.

Hopefully this makes sense and is a system we can use going forward?

File: docs/scenarios/README.md [hunk 5]
```diff
@@ -20,9 +19,48 @@ Scenarios are divided into one or more steps. Each step has an image, scenario d
...
+**start/end positions:** North `N`, East `E`, South `S`, West `W`, static in lane `St`, double parked `DP`, parked on roadside `Pa`, driveway `Dr`, etc <br>
+**action:** `01`, `02`, `03`, etc. These "action" options are only used if the actor's action is out of the ordinary or isn't obvious, and they do not carry over between scenario categories. For example, 01 for the CF scenarios (sudden stop of a lead vehicle) translates to a different action than 01 for the I scenarios (Ego has right of way at stop sign).
```
[the remark this answers is not in the document]
Comment (greenmustard) [reply docs/scenarios/README.md]: These action IDs will have their own key/map at the top of the scenario category pages.

File: docs/scenarios/intersections.md [hunk 6]
```diff
@@ -44,43 +19,44 @@ The following are examples on how these various configurations can be put togeth
...
 |      |     |     |
 | :--: | --- | --- |
 | A    | ![I-02-A](./images/I-02-A.png) | **Scenario** <br> Ego approaches the intersection. Secondary vehicle approaches from South and stops. <br><br> **Expected Result** <br> Ego comes to a stop in front of the stopline. |
-| B    | ![I-02-B](./images/I-02-B.png) | **Scenario** <br> Secondary vehicle turns left <br><br> **Expected Result** <br> Ego remains stopped. |:
+| B    | ![I-02-B](./images/I-02-B.png) | **Scenario** <br> Secondary vehicle turns left <br><br> **Expected Result** <br> Ego remains stopped. |
 | C    | ![I-02-C](./images/I-02-C.png) | **Scenario** <br> Secondary vehicle clears the intersection. <br><br> **Expected Result** <br> Ego proceeds through the intersection. |
 | D    | ![I-02-D](./images/I-02-D.png) | **Scenario Complete 🎉** |
 
-## 3-Way uncontrolled intersection with another vehicle
+**3-2-NSW-I-L-CAR:S>W:02**
```
[the remark this answers is not in the document]
Comment (greenmustard) [reply docs/scenarios/intersections.md]: All the actors should be white now.

**Comment by Eric-Gonzalez** [comment]
:tada: Awesome work @greenmustard :shipit:

**Pull request merged**